docs/how-to/installation/packages.txt

```diff
@@ -5,14 +5,76 @@ Install Determined Using Debian Packages
 
 For systems running Ubuntu 16.04 or 18.04, we support installing the
 Determined master and agent using Debian packages and running them as
-systemd services rather than Docker containers.
-
-For PostgreSQL, you should use either your Linux distribution's package
-and service or a :ref:`Docker container <install-using-docker>`.
+systemd services.
 
 Preliminary Setup
 -----------------
 
+PostgreSQL
+~~~~~~~~~~
+Determined leverages a PostgreSQL database to store experiment and trial
+metadata. You may either leverage your Linux distribution's package and service
+or a Docker container.
+
+With either of the following installation methods, PostgreSQL will typically run
+with a default of 100 ``max_connections``, which is sufficient for Determined.
+You may validate by issuing the following query via a database client like
+``psql``:
+
+.. code::
+
+   SHOW max_connections;
+
+
+Installing PostgreSQL via ``apt``
+_________________________________
+We recommend installing the following version of PostgreSQL:
+
+.. code::
+
+   sudo apt install postgresql-10
+
+Next, configure a system account that Determined will use to connect to
+PostgreSQL. For example, to use the default ``postgres`` user but update its
+password:
+
+.. code::
+
+   sudo -u postgres psql postgres
+   postgres=# \password postgres
+
+Finally, create a database for Determined's use:
+
+.. code::
+
+   postgres=# CREATE DATABASE determined;
+
+Running PostgreSQL in Docker
+____________________________
+Pull the official Docker image for PostgreSQL. We recommend using the version
+listed below.
+
+.. code::
+
+   docker pull postgres:10
+
+This image is not provided by Determined AI; please see `its Docker Hub page
+<https://hub.docker.com/_/postgres>`_ for more information.
+
+Start PostgreSQL as follows:
+
+.. code::
+
+   docker run \
+       -d \
+       --restart unless-stopped \
+       --name determined-db \
+       -p 5432:5432 \
+       -v determined_db:/var/lib/postgresql/data \
+       -e POSTGRES_DB=determined \
+       -e POSTGRES_PASSWORD=<Database password> \
+       postgres:10
+
```

Comment:
Add the same note here as in the other doc about the option to use `-p 127.0.0.1:5432:5432`.

Reply:
Done